vendor/modules.txt

```diff
@@ -419,7 +419,6 @@ github.com/go-openapi/jsonreference
 # github.com/go-openapi/loads => github.com/go-openapi/loads v0.19.4
 # github.com/go-openapi/runtime => github.com/go-openapi/runtime v0.19.4
 # github.com/go-openapi/spec v0.19.5 => github.com/go-openapi/spec v0.19.5
-## explicit
 github.com/go-openapi/spec
```

Comment:
we didn't actually drop anything under `vendor/github.com/go-openapi/spec` as part of this, so we actually increased our dependencies... what is requiring `github.com/go-openapi/spec` now?

Reply:
@liggitt this is what I shared in Slack:

`github.com/go-openapi/spec` is still being pulled in by `kustomize` though (but k/k doesn't depend on it anymore).

```console
$ go mod why github.com/go-openapi/spec
# github.com/go-openapi/spec
k8s.io/kubernetes/pkg/kubectl/cmd/convert
k8s.io/cli-runtime/pkg/resource
sigs.k8s.io/kustomize/api/krusty
sigs.k8s.io/kustomize/kyaml/openapi
github.com/go-openapi/spec
```

Not sure if that's something we want to change in the future (https://github.com/kubernetes-sigs/kustomize/blob/f93cee94404bf0b60ff82785be9ff2af9797463f/kyaml/openapi/openapi.go#L14).

Reply:
ah, right... https://github.com/kubernetes/kubernetes/pull/98946#discussion_r589798392

tracked in https://github.com/kubernetes-sigs/kustomize/issues/3705